soc/arm/ti_simplelink/cc13x2_cc26x2/Kconfig.series

```diff
@@ -0,0 +1,15 @@
+# Kconfig.series - Texas Instruments SimpleLink CC13x2 / CC26x2
+#
+# Copyright (c) 2019 Brett Witherspoon
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_SERIES_CC13X2_CC26X2
+	bool "TI SimpleLink Family CC13x2 / CC26x2"
+	select CPU_CORTEX_M4
+	select SOC_FAMILY_TISIMPLELINK
```

Comment:
If this is a CORTEX-M4F, with an FPU, wondering if we want to enable FPU and add:
`select CPU_HAS_FPU`
here?

Reply:
Yes, this is a Cortex-M4F so CPU_HAS_FPU makes sense. There is also an MPU so we might want to enable CPU_HAS_MPU.

Reply:
I'm guessing enabling FPU may save some footprint if doing any floating point.
Enabling MPU may have bigger consequences: may want to check the code to see what Zephyr will do/require in those cases.

Reply:
I'll let the MPU be enabled as needed in future PRs.